go.mod

```
@@ -1,19 +1,33 @@
 module pmm-dump
 
-go 1.16
+go 1.19
```

```suggestion
go 1.20
```

Let's use the latest go version